src/Workspaces/CSharp/Portable/Editing/CSharpImportAdder.cs

```diff
@@ -28,6 +30,16 @@ protected override INamespaceSymbol GetExplicitNamespaceSymbol(SyntaxNode node,
             return null;
         }
 
+        protected override INamespaceSymbol GetContainedNamespace(SyntaxNode node, SemanticModel model)
```

Comment:
nit: consider naming TryGetContainedNamespace to plainly indicate it can return null